docs/architecture/adr-069-gov-improvements.md

```diff
@@ -0,0 +1,200 @@
+
+```protobuf
+enum ProposalType {
+  NORMAL = 0;
+  MULTIPLE_CHOICE = 1;
+  OPTIMISTIC = 2;
+  EXPEDITED = 3;
+}
+```
+
+Note, that expedited becomes a proposal type itself instead of a boolean on the `v1.Proposal` struct.
+
+> An expedited proposal is by design a normal proposal with a quicker voting period and higher threshold. When an expedited proposal fails, it gets converted to a normal proposal.   
+
+An expedited optimistic proposal and an expedited multiple choice proposal do not make sense based on the definition above and is a proposal type instead of a proposal characteristic.
+
+#### Optimistic Proposal
+
+An optimistic proposal is a proposal that passes unless a threshold a NO votes is reached.
+
+Voter can only vote NO on the proposal. If the NO threshold is reached, the optimistic proposal is converted to a normal proposal.
+
+Two governance parameters will be in added [`v1.Params`][5] to support optimistic proposals:
+
+```protobuf
+// optimistic_authorized_addreses is an optional governance parameter that limits the authorized accounts than can submit optimisitc proposals
+repeated string optimistic_authorized_addreses = 17 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+// Optimistic rejected threshold defines at which percentage of NO votes, the optimistic proposal should fail and be converted to a normal proposal.
+string optimistic_rejected_threshold = 18 [(cosmos_proto.scalar) = "cosmos.Dec"];
+```
+
+#### Multiple Choice Proposal
+
+A multiple choice proposal is a proposal where the voting options can be defined by the proposer.
+
+The number of voting option will be limited to a maximum of 4. A new vote option `SPAM` will be added and distinguished of those voting options. 
+
+Multiple choice proposals, contrary to any other proposal type, cannot have messages to execute. They are only text proposals.
+
+Submitting a new multiple choice proposal will use a different message than the [`v1.MsgSubmitProposal`][5]. This is done in order to simplify the proposal submittion and allow defining the voting options directly.
+
+
+```protobuf
+message MsgSubmitMultipleChoiceProposal {
+  repeated cosmos.base.v1beta1.Coin initial_deposit = 1
+  string proposer = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string metadata = 3;
+  string title = 4;
+  string summary = 5;
+  string option_one = 6;
+  string option_two = 7;
+  string option_three = 8;
+  string option_four = 9;
+}
+```
+
+Voters can only vote on the defined options in the proposal.
+
+To maintain compatibility with the existing endpoints, the voting options will not be stored in the proposal itself and each option will be mapped to [`v1.VoteOption`][5]. A multiple choice proposal will be stored as a [`v1.Proposal`][5]. A query will be available for multiple choice proposal types to get the voting options.
+
+Multiple choice proposals do not need additional governance parameters.
```

Comment:
Is this still true with the recent addition of the `burn_spam_amount` param?

Reply:
good one ty!